tools/bumper/cnaoRepoCommands.go

```diff
@@ -166,3 +191,20 @@ func isVtagFormat(tagVersion string) bool {
 	var vtagSyntax = regexp.MustCompile(`^[0-9]\.[0-9]+\.*[0-9]*-[0-9]+-g[0-9,a-f]{7}`)
 	return vtagSyntax.MatchString(tagVersion)
 }
+
+func runExternalGitCommand(args []string) error {
+	// TODO replace this when go-git adds advanced checkout/restore/clean abilities so we could checkout specific paths
+	cmd := exec.Command("git", args...)
+	cmd.Stderr = os.Stderr
+	cmd.Stdout = os.Stdout
+	err := cmd.Run()
+	if err != nil {
+		return errors.Wrapf(err, "Failed to run git command: git %s", strings.Join(args, " "))
```

Comment:
Can we add Stderr and Stdout to the returned error ?

Reply:
Done